data/data-pipeline/data_pipeline/score/field_names.py

```diff
@@ -200,30 +204,63 @@
 HOLC_GRADE_D_TRACT_75_PERCENT_FIELD: str = "Tract is >75% HOLC Grade D"
 
 
+# Child Opportunity Index data
+# Summer days with maximum temperature above 90F.
+EXTREME_HEAT_FIELD = "Summer days above 90F"
+
+# Percentage households without a car located further than a half-mile from the
+# nearest supermarket.
+HEALTHY_FOOD_FIELD = "Percent low access to healthy food"
+
+# Percentage impenetrable surface areas such as rooftops, roads or parking lots.
+IMPENETRABLE_SURFACES_FIELD = "Percent impenetrable surface areas"
+
+# Percentage third graders scoring proficient on standardized reading tests,
+# converted to NAEP scale score points.
+READING_FIELD = "Third grade reading proficiency"
+LOW_READING_FIELD = "Low third grade reading proficiency"
+
+# Names for individual factors being exceeded
 # Climate Change
 EXPECTED_POPULATION_LOSS_RATE_LOW_INCOME_FIELD = f"At or above the {PERCENTILE}th percentile for expected population loss rate and is low income"
 EXPECTED_AGRICULTURE_LOSS_RATE_LOW_INCOME_FIELD = f"At or above the {PERCENTILE}th percentile for expected agriculture loss rate and is low income"
 EXPECTED_BUILDING_LOSS_RATE_LOW_INCOME_FIELD = f"At or above the {PERCENTILE}th percentile for expected building loss rate and is low income"
+EXTREME_HEAT_MEDIAN_HOUSE_VALUE_LOW_INCOME_FIELD = (
+    f"At or above the {PERCENTILE}th percentile for summer days above 90F and "
+    f"the median house value is less than {MEDIAN_HOUSE_VALUE_PERCENTILE}th "
+    f"percentile and is low income"
+)
 
 # Clean energy and efficiency
 PM25_EXPOSURE_LOW_INCOME_FIELD = f"At or above the {PERCENTILE}th percentile for PM2.5 exposure and is low income"
 ENERGY_BURDEN_LOW_INCOME_FIELD = f"At or above the {PERCENTILE}th percentile for energy burden and is low income"
+
 # Clean transportation
 DIESEL_PARTICULATE_MATTER_LOW_INCOME_FIELD = f"At or above the {PERCENTILE}th percentile for diesel particulate matter and is low income"
 TRAFFIC_PROXIMITY_LOW_INCOME_FIELD = f"At or above the {PERCENTILE}th percentile for traffic proximity and is low income"
+
 # Affordable and Sustainable Housing
-LEAD_PAINT_MEDIAN_HOME_VALUE_LOW_INCOME_FIELD = (
+LEAD_PAINT_MEDIAN_HOUSE_VALUE_LOW_INCOME_FIELD = (
```

Comment:
Just standardizing on using the word 'house' instead of 'home'.

Reply:
Thanks